### PR TITLE
Fix/ie11 es6 issue

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Currently IE11 breaks since the `locales` folder has been moved out of `/src`

# Solution
Include `/locales` into `/src`. It makes more sense to keep the locales within the source and it will also make life easier because the code can be maintained using ES6 syntax.
 
## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-301-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 ### Added
 - Public: Added support for Spanish language. The SDK can now be used in Spanish by adding `{language: 'es'}` to the initialization options. If `language` is not present or the wrong locale tag is provided, the language locale will default to `en`.
 - Public: Added support for Spanish language on the SMS body.
+- Internal: Moved `/locales` folder within `src`
```
Comment (rfreitas) [inline CHANGELOG.md]: not sure if we need to put it on the changelog. was there a locales folder on the previous version? did it break on IE then?

Comment (stefaniacardenas) [reply CHANGELOG.md]: There wasn't a `/locales` folder in previous versions. I agree, no need for changelog.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-301-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://301-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**